how-to/how-to-use-kata-containers-with-nemu.md

```diff
@@ -0,0 +1,120 @@
+
+# Kata Containers with NEMU
+
+* [How to get started](#how-to-get-started)
+    * [Pre-requisites](#pre-requisites)
+    * [NEMU](#nemu)
+        * [Download, build and install](#download-build-and-install)
+    * [Configure Kata Containers](#configure-kata-containers)
+
+Kata Containers relies by default on the QEMU hypervisor in order to spawn the virtual machines running containers. [NEMU](https://github.com/intel/nemu) is a fork of QEMU that:
+- Reduces the number of lines of code.
+- Removes all legacy devices.
+- Reduces the emulation as far as possible.
+
+## How to get started
+
+This section describes how to run Kata Containers with NEMU, first by explaining how to download, build and install it. Then this document walks through the steps needed to update your Kata Containers configuration in order to run with NEMU. 
+
+### Pre-requisites
+This document requires Kata Containers to be [installed](https://github.com/kata-containers/documentation/blob/master/install/README.md) on your system.
+
+Also, it's worth noting that NEMU only supports `x86` and `ARM` architecture.
+
+### NEMU
+
+#### Download, build and install
+
+```bash
+$ git clone https://github.com/intel/nemu.git
+$ cd nemu
+$ git fetch origin/master
+$ git checkout experiment/automatic-removal
+```
+__x86__
+```
+$ SRCDIR=$PWD ./tools/build_x86_64_virt.sh
+```
+__ARM__
+```
+$ SRCDIR=$PWD ./tools/build_aarch64.sh
+```
+
+___Note:__ The branch `experiment/automatic-removal` is a branch published by Jenkins after it applied the automatic removal script to the `topic/virt-x86` branch. The purpose of this code removal being to reduce the source tree by removing files not being used by NEMU._
+
+After those commands successfully returned, you will find the NEMU binary at `$HOME/build-x86_64/x86_64_virt-softmmu/qemu-system-x86_64_virt` (__x86__), or `$HOME/build-aarch64/aarch64-softmmu/qemu-system-aarch64` (__ARM__).
```

Comment:
For `x86_64`, the path that was created on my machine is: `/home/foo/build-x86_64_virt/x86_64_virt-softmmu/qemu-system-x86_64_virt`

Reply:
@chavafg That is correct, the binary will be under `~/build-x86_64_virt` and not `~/build-x86_64`.

Reply:
> I think this would be clearer as a list:
> 
> > After a successful build, he built NEMU binary will be available at:
> > 
> > * `x86_64`: `$HOME/build-x86_64/x86_64_virt-softmmu/qemu-system-x86_64_virt`
> > * `ARM64`: `$HOME/build-aarch64/aarch64-softmmu/qemu-system-aarch64`
> 
> BTW, is it correct that the ARM64 binary name does not contain "`virt`"?

Yes, that is correct.